terraform-local-docker/variables.tf

```diff
@@ -0,0 +1,24 @@
+variable "MYSQL_ROOT_PASSWORD" {
+  type = string
```

Comment:
You could also add  sensitive and maybe a description
When a variable is sensitive it is not shown in the log files
```
variable "db_password" {
  description = "Database administrator password"
  type        = string
  sensitive   = true
}
```